scripts/ogr-convert.sh

```diff
@@ -57,17 +57,36 @@ if [ "$source_mime" == "application/zip" ]; then
   mkdir "${DATA_DIR}/source-zip" || true
   unzip "$source_loc" -d "${DATA_DIR}/source-zip"
 
-  # count extracted files
-  archive_file_count=$(ls -1 "${DATA_DIR}/source-zip/" | wc -l)
-
-  if [ $archive_file_count -eq 1 ]; then
-    # single file -> use as new source
-    extracted_file=$(ls -1 "${DATA_DIR}/source-zip/")
-    source_loc="${DATA_DIR}/source-zip/$extracted_file"
-  else
-    # multiple files
+  # Handle case of nested Zip files (hale-connect Shapefile original source for Atom feed)
+  # if all files within source_loc are .zip files, extract them
+  if [ "$(ls -1 ${DATA_DIR}/source-zip/*.zip 2>/dev/null | wc -l)" -gt 0 ]; then
+    echo "Detected only .zip files in source folder, extracting them"
```

Comment:
> Detected only .zip files in source folder

On first glance, the `if` conditions appears to be true also in case there are files with other extensions than `.zip` in the archive. Such files appear not to be processed further. Is that intentional? If yes, I would probably reword the log message slightly.

Reply:
Thanks for the hint, lost track of this when refactoring the script while working on it. Will look into how to check if there are only `.zip` files.

Reply:
Updated the logic